### ISSUE TITLE
build make-meds script

### ISSUE THREAD

**Issue by beckermr** [esheldon/meds#12]
It should be called like this

```make-meds --meta-file {metafile} -c {configfile} obj_data_file image_info_file
make-meds-from-stubby stubbyfile
make-meds --from-stubby-file stubbyfile
```

In the last case, it passes in config from the stubby file through the `conf` keyword of the `MEDSMaker`.

**Comment by beckermr** [comment]
It looks like some of this stuff made it into esheldon/desmeds. Do we still want these scripts here?

**Comment by esheldon** [comment]
Yes, I do want a script in meds called `make-meds` but I haven't had time to implement it yet.
